draft-ietf-ppm-dap.md

```diff
@@ -1153,7 +1134,12 @@ struct {
 ~~~
 
 Intervals of time consist of a start time and a duration. Intervals are
-half-open; that is, `start` is included and `(start + duration)` is excluded.
+half-open; that is, `start` is included and `(start + duration)` is excluded. A
+time that is before an `Interval`'s `start` is said to be before that interval.
+A time that is equal to or after `Interval.start + Interval.duration` is said to
+be after the interval. A time that is either before or after an interval is said
+to be outside the interval. A time that is neither before nor after an interval
+is said to be inside or fall within the interval.
```

Comment:
This extra-pedantic definition is useful later, when we discuss the conditions under which aggregators should ignore reports.